src/utils/image_server.cljs

```diff
@@ -299,3 +210,24 @@
                                     "&size="
                                     qr-size)]
     media-server-url))
+
+(defn get-image-uri
+  [{:keys [type options]}
+   profile-picture-options]
+  ((case type
+     :account  get-account-image-uri
+     :contact  get-contact-image-uri
+     :initials get-initials-avatar-uri
+     str)
+   (-> (merge options profile-picture-options)
```

Comment:
just wondering why we can't merge outside this function?

Reply:
I suppose we could merge on the outside, but at the moment `merge` is sorta an internal detail atm. For example, later on we could change how we extract options from each of the maps.